client/src/main/java/com/vesoft/nebula/client/graph/net/Session.java

```diff
@@ -260,4 +364,98 @@ private boolean retryConnect() {
             return false;
         }
     }
+
+    /**
+     * convert java list to nebula thrift list
+     * @param list java list
+     * @return nebula list
+     */
+    private NList list2Nlist(List<Object> list) {
+        NList nlist = new NList();
+        try {
+            for (Object item : list) {
+                nlist.values.add(value2Nvalue(item));
+            }
+        } catch (UnsupportedOperationException e) {
+            throw e;
```

Comment:
if just catch the exception and then throw it, no need to try.

Reply:
ACK.